modules/distr-tracing-install-jaeger-operator.adoc

```diff
@@ -5,40 +5,35 @@ This module included in the following assemblies:
 
 .Prerequisites
 * You have access to the {product-title} web console.
 * You have access to the cluster as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
-* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {JaegerName} Operator.
-
-[WARNING]
-====
-Do not install Community versions of the Operators. Community Operators are not supported.
-====
+* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {JaegerOperator} Operator.
 
 .Procedure
 
 . Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
 
 . Navigate to *Operators* -> *OperatorHub*.
 
-. Type *distributed tracing platform* into the filter to locate the {JaegerName} Operator.
+. Search for the {JaegerOperator} Operator by entering *distributed tracing platform* in the search field.
 
-. Click the *{JaegerName} Operator* provided by Red Hat to display information about the Operator.
+. Select the *{JaegerOperator}* Operator, which is *provided by Red Hat*, to display information about the Operator.
 
 . Click *Install*.
 
-. On the *Install Operator* page, select the *stable* Update Channel. This automatically updates your Operator as new versions are released.
+. On the *Install Operator* page, select the *stable* for the *Update channel*. This will automatically update the Operator when new versions are released.
 //If you select a maintenance channel, for example, *Stable*, you will receive bug fixes and security patches for the length of the support cycle for that version.
 
 . Accept the default *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and makes the Operator available to all projects in the cluster.
 
-* Accept the default *Automatic* approval strategy. By accepting the default, when a new version of this Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator without human intervention. If you select *Manual* updates, when a newer version of an Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.
+. Accept the default *Automatic* approval strategy. By accepting the default, when a new version of this Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator without human intervention. If you select *Manual* updates, when a newer version of an Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.
```

Comment:
```suggestion
. Accept the default *Automatic* approval strategy. If you accept the default, when a new version of this Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator without human intervention. If you select *Manual* updates, when a newer version of an Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.
```

Reply:
"By accepting the default" looks like it's OLM that accepts the default.

Reply:
Indeed. Fixed. Thank you.